### PR TITLE
Print invoices, Attributes for customers..

### PR DESCRIPTION
**Pull request by angoyd**
Added printinvoice and attributes for customers, some fields added in inventorysummary and changed the way to get sales orders with the new Ordertype in the request..

### CODE REVIEW

**Comment by omelhus** [comment]
Hey! Thanks for this! 

Can you pull the dev repository and fix the merge issues? If you can just do that I'll be happy to accept the pull request.

**Comment by angoyd** [comment]
@omelhus merged with upstream and solved the conflicts.

File: Visma.net/Models/InventorySummary.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿using ONIT.VismaNetApi.Lib;
+using ONIT.VismaNetApi.Models.CustomDto;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONIT.VismaNetApi.Models
+{
+    public class InventorySummary : DtoProviderBase, IProvideIdentificator
```
Comment (omelhus) [inline Visma.net/Models/InventorySummary.cs]: If it doesn't provide an identificator it shouldn't inherit IProvideIdentificator

File: Visma.net/Models/InventorySummary.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        public decimal dropShipForSOPrepared { get; set; }
+        public decimal dropShipForSOReceipts { get; set; }
+
+        public decimal estimatedUnitCost { get; set; }
+        public decimal estimatedTotalCost { get; set; }
+
+        public string baseUnit { get; set; }
+        public string GetIdentificator()
+        {
+            throw new NotImplementedException();
```
Comment (omelhus) [inline Visma.net/Models/InventorySummary.cs]: I don't think this is a good idea. It's better to remove the function.

File: ONIT.VismaNet/ONIT.VismaNet.csproj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (omelhus) [inline ONIT.VismaNet/ONIT.VismaNet.csproj]: This file must be updated to the most current one from the repository.
```xml
<Project Sdk="Microsoft.NET.Sdk">
    <PropertyGroup>
        <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
	<Title>Visma.net</Title>
        <Version>3.0.3</Version>
        <Copyright>Copyright © ON IT AS 2014 - 2017</Copyright>
        <Description>Get started with Visma.net Integrations.</Description>
        <Authors>ON IT AS</Authors>
        <Company>ON IT AS</Company>
        <PackageLicenseUrl>https://github.com/ON-IT/Visma.Net/blob/master/LICENSE.md</PackageLicenseUrl>
        <PackageProjectUrl>https://github.com/ON-IT/Visma.Net</PackageProjectUrl>
        <RepositoryUrl>https://github.com/ON-IT/Visma.Net.git</RepositoryUrl>
        <PackageTags>Visma.net API, Visma.net Integrations, REST</PackageTags>
        <PackageReleaseNotes>Fixed an issue with setting addressLine1 to null on Address.</PackageReleaseNotes>
        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
        <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
        <RepositoryType>git</RepositoryType>
        <RootNamespace>ONIT.VismaNetApi</RootNamespace>
        <PackageIconUrl>https://sync-it.no/Images/on-it_logo.png</PackageIconUrl>
        <AssemblyVersion>3.0.3.0</AssemblyVersion>
        <FileVersion>3.0.3.0</FileVersion>
    </PropertyGroup>
    <ItemGroup>
      <PackageReference Include="Microsoft.CSharp" Version="4.4.0" />
      <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
      <PackageReference Include="System.Net.Http" Version="4.3.3" />
    </ItemGroup>
</Project>
```

**Comment by angoyd** [comment]
Now I think I got it all.

**Pull request merged**